test/instrumentation/_agent.js

```diff
@@ -12,27 +13,14 @@ var sharedInstrumentation
 
 module.exports = function mockAgent (expected, cb) {
   var agent = {
-    _conf: {
-      serviceName: 'service-name',
-      active: true,
-      instrument: true,
-      captureSpanStackTraces: true,
+    _conf: config({
+      abortedErrorThreshold: '250ms',
+      asyncHooks: false,
```

Comment:
Why did you add this?

Reply:
There was a test issue...I forget exactly what it broke. Technically though, the _omission_ of `asyncHooks: false` previously made it _implicitly_ false and this just restores the previous behaviour.

Reply:
As this is a mock agent I'm just wondering where this makes a difference... and since you didn't add any new tests, it's weird that this update should change anything. So I'm a bit worried if it's covering over something else 🤔